### PR TITLE
fix(learn): Organized the Certifications on Settings to match the main page

### LINKED ISSUE #39129: [beta] Infosec cert out of order on settings page
The map has the certifications in this order:
```
Responsive Web Design Certification
JavaScript Algorithms and Data Structures Certification
Front End Libraries Certification
Data Visualization Certification
APIs and Microservices Certification
Quality Assurance Certification
Scientific Computing with Python Certification
Data Analysis with Python Certification
  Information Security Certification
Machine Learning with Python Certification
```

But the settings page has the certs in this order:
```
Responsive Web Design Certification
JavaScript Algorithms and Data Structures Certification
Front End Libraries Certification
Data Visualization Certification
APIs and Microservices Certification
Quality Assurance Certification
  Information Security Certification
Scientific Computing with Python Certification
Data Analysis with Python Certification
Machine Learning with Python Certification
```

Pretty minor, but the `infosec` cert is in a different spot on the settings page. It could be changed to align with the map.

### PR DESCRIPTION
**Pull request by JovaniTarnowski**
Fixes freeCodeCamp/freeCodeCamp#39129

Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `master` branch of freeCodeCamp.
- [X] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #39129

### CODE REVIEW

**Comment by RandellDawson** [comment]
@ojeytonwilliams @raisedadead Do we really want to have to maintain the same data in two places?  Would it not be better to have this information pulled from a common place?

**Comment by ojeytonwilliams** [comment]
> Do we really want to have to maintain the same data in two places? Would it not be better to have this information pulled from a common place?

Yep, it would be better.  However, I think that refactor should wait until after beta has been pushed to production, since the beta branch has been QA'd a fair bit and we want to make sure the launch is a smooth as possible.

**Review by PeterF521 — DISMISSED** [review]

File: client/src/resources/certAndProjectMap.js [hunk 1]
```diff
@@ -618,6 +580,83 @@ const certMap = [
...
+        superBlock: 'information-security-v7'
+      },
+      {
+        id: '5e601c775ac9d0ecd8b94aff',
+        title: 'Secure Real Time Multiplayer Game',
+        link: `${infoSecBase}/secure-real-time-multiplayer-game`,
+        superBlock: 'information-security-v7'
+      }
+    ]
+  },
```
Comment (ojeytonwilliams) [inline client/src/resources/certAndProjectMap.js:659]:
```suggestion
```
This copy was redundant.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
It works nicely, but the second copy should go.

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
Thanks for sorting this out, @JovaniTarnowski, much appreciated!

**Pull request merged**